manually-managed/README.md

```diff
@@ -0,0 +1,33 @@
+# Manually Managed Services
+
+While most of our infrastructure is managed with Kubernetes, several stateful services and other components are managed separately. At current, these are managed manually, but we are considering the best tools for this, e.g. Ansible. (If you have expertise or time to contribute here, let us know in the issues or on Slack!)
+
+
+## Redis Cache
+
+We use Redis for caching and for queuing in Rails (that is, in `web-monitoring-db`). Our Redis caching instance is currently hand-managed as an EC2 machine on AWS. Its current configuration (excepting the password) can be found in [`redis-cache.conf`](./redis-cache.conf). It also has an elastic IP.
+
+The VM and other related resources (e.g. Elastic IP) are named `wm-cache-*`.
+
+This should *probably* be a cluster (and maybe managed via AWS Elasticache?), but this has been working alright for now.
+
+
+## Redis Queues
+
+The asynchronous jobs that are part of the `web-monitoring-db` codebase (for importing data, sending e-mails, and auto-analyzing versions) are based on Redis. At current, they are managed within Kubernetes, but that means they are prone to losing their state. These should be moved to AWS Elasticache or some manually managed system in the future (we may also move off Redis for our queues, too).
+
+TBD
+
+
+## Postgres via RDS
+
+Web-monitoring-db’s database is a PostgreSQL database managed via RDS.
```

Comment:
Use backticks for `Web-monitoring-db` to match formatting in previous sections. (maybe lower-case, too? /shrug Don't think we ever created a branding guide to know for sure =P)

Reply:
Actually, I think none of these should have had backticks. But I take your point about consistency.

Reply:
Sounds good to me. Yea, main point was being consistent